src/components/application_manager/rpc_plugins/rc_rpc_plugin/include/rc_rpc_plugin/rc_module_constants.h

```diff
@@ -44,6 +44,7 @@ const char kbuttonCapabilities[] = "buttonCapabilities";
 
 const char kRadioControlData[] = "radioControlData";
 const char kClimateControlData[] = "climateControlData";
+const char msg_params[] = "msg_params";
```

Comment:
@LitvinenkoIra  please use constants from application manager if it is possible.